projects/components/src/table/cells/data-renderers/relative-timestamp/relative-timestamp-table-cell-renderer.component.ts

```diff
@@ -0,0 +1,63 @@
+import { ChangeDetectionStrategy, Component, Inject, OnInit } from '@angular/core';
+import { DateCoercer, DateFormatMode, DateFormatOptions } from '@hypertrace/common';
+import { TableColumnConfig } from '../../../table-api';
+import {
+  TABLE_CELL_DATA,
+  TABLE_COLUMN_CONFIG,
+  TABLE_COLUMN_INDEX,
+  TABLE_DATA_PARSER,
+  TABLE_ROW_DATA
+} from '../../table-cell-injection';
+import { TableCellParserBase } from '../../table-cell-parser-base';
+import { TableCellRenderer } from '../../table-cell-renderer';
+import { TableCellRendererBase } from '../../table-cell-renderer-base';
+import { CoreTableCellParserType } from '../../types/core-table-cell-parser-type';
+import { CoreTableCellRendererType } from '../../types/core-table-cell-renderer-type';
+import { TableCellAlignmentType } from '../../types/table-cell-alignment-type';
+
+export interface RowData {
```

Comment:
nit: You can do 
```
export interface RowData extends Dictionary<unknown> {
baseTimestamp: Date | number;
}
```

Reply:
Also, can the timestamp be string too?

Reply:
I guess, `Date` can handle the `string`, I have tested this, works fine

custom type `DateOrNumber` is used as type for celldata as well so I didn't want to repeat `Date | number` so created this type.

`extends Dictionary<unknown> ` part is fixed!!